<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>560</width>
     <height>270</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btntoHome">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>332</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
    </font>
   </property>
   <property name="text">
    <string>홈으로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInfo">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
    </font>
   </property>
   <property name="text">
    <string>전체 조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
